Resources/Private/PlaceholderInsert/src/PlaceholderInsertDropdown.js

```diff
@@ -74,4 +69,33 @@ export default class PlaceholderInsertDropdown extends PureComponent {
       />
     );
   }
+
+  getOptionsRecursively(elements) {
+    const {nodeTypeRegistry} = this.props;
+    const returnValues = [];
+
+
+    elements.forEach((element) => {
+      const node = this.props.nodesByContextPath[element.contextPath];
+      const childNodes = this.props.nodesByContextPath[element.contextPath].children;
+
+      if (nodeTypeRegistry.isOfType(node.nodeType, 'Neos.Form.Builder:Mixin.PlaceholderInsert')) {
```

Comment:
this wont work right away, due to the optimisations to the schema: https://github.com/neos/neos-ui/issues/3677

Reply:
I fixed this. It is required to add the nodetypes to the frontend configuration and it is possible to exclude all child nodes of a specific node type (which is useful for the options and validator collection because it is not required to list them all)